src/modelfree/common/utils.py

```diff
@@ -59,15 +60,23 @@ def __init__(self, policy):
         """
         super().__init__(policy=policy, sess=policy.sess)
 
-    def predict(self, observation, state=None, mask=None, deterministic=False):
+    def predict(self, observation, state=None, mask=None, deterministic=False, return_data=True):
```

Comment:
Seems useful to have some kind of docstring or general comment here clarifying what happens when return_data is set to true (and the fact that it has no effect unless used on a Transparent policy), especially since `data` is a pretty general term